charts/prometheus-blackbox-exporter/templates/_helpers.tpl

```diff
@@ -150,10 +150,10 @@ securityContext:
 {{- end }}
 {{- with .Values.extraInitContainers }}
 initContainers:
-  {{- toYaml . | indent 2 }}
+{{ toYaml . | indent 2 }}
 {{- end }}
 containers:
-{{- with .Values.extraContainers }}
+{{ with .Values.extraContainers }}
```

Comment:
imho that minus can stay

Reply:
If the minus sign is not removed, it will result in a template rendering error.